package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "dropdown2",
   "version": "0.1.0",
-  "homepage": "https://raquel-ucl.github.io/plot-explorer-clean/",
+  "homepage": "https://github-pages.ucl.ac.uk/UKCORDEX-plot-explorer/",
   "private": true,
   "dependencies": {
     "@testing-library/jest-dom": "^5.11.4",
```

Comment:
npm added `"bootstrap": "^5.1.3",` to my dependencies list when I managed to get this working. I'm guessing it should be added here as part of this PR

Reply:
That's in the next PR https://github.com/UCL/UKCORDEX-plot-explorer/pull/12/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R10, but I can cherry-pick https://github.com/UCL/UKCORDEX-plot-explorer/commit/921b1826b233710fbc09fdc8f04b17ae5de15733#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519 into this branch.